dehoister/dehoister.py

```diff
@@ -9,16 +9,19 @@
 from redbot.core.utils.menus import start_adding_reactions
 from redbot.core.utils.predicates import ReactionPredicate
 
+from typing import Union
+
 log = logging.getLogger("red.kreusada.dehoister")
 
 IDENTIFIER = 435089473534
 
-HOIST = "!\"#$%&'()*+,-./:;<=>?@"
+HOIST = tuple("!\"#$%&'()*+,-./:;<=>?@")
+# Since every time this got used it was converted to a tuple just make a tuple here
```

Comment:
```suggestion
```
We can remove this comment here, thanks for letting me know about it though, makes sense.